src/test/scala/com/nec/cmake/DynamicCSqlExpressionEvaluationSpec.scala

```diff
@@ -933,6 +935,17 @@ class DynamicCSqlExpressionEvaluationSpec
     }
   }
 
+  s"Timestamps are supported" in withSparkSession2(configuration) { sparkSession =>
+    import sparkSession.implicits._
+
+    val before = Instant.now();
+    val sql =
+      "select max(b) from values (1, NOW()), (2, NOW()), (3, NOW()) as tab1(a, b)"
+    sparkSession.sql(sql).debugSqlHere { ds =>
+      assert(ds.as[Instant].collect().toList.head > before)
```

Comment:
might be good to give some sort of an upper bound, even if it's 1hr.

Reply:
Technically if you ran the code at exactly 1am on a daylight savings time switch it could be up to 1 hour different... and fail completely the other way around.